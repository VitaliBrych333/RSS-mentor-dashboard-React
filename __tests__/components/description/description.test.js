import React from 'react';
import { shallow, configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import Description from '../../../src/components/description';

configure({ adapter: new Adapter() });

describe('Description ', () => {
    test('match snapshot', () => {
        const wrapper = shallow(<Description />);
        expect(wrapper).toMatchSnapshot();
    });
});
