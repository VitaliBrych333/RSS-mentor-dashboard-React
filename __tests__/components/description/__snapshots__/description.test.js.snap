// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description  match snapshot 1`] = `""`;
